### DISCUSSION TITLE
Stats card and streaks  are not shown in my profile

### DISCUSSION THREAD

**Discussion by klpranaav** [anuraghazra/github-readme-stats#3617] [Q&A] [answered]
### Describe the bug

My Stat cards and streaks  are not showing on my profile: https://github.com/klpranaav. Cards are shown correctly when I edit the readme locally. They are also shown correctly in this message and if I open the link on a new tab.

### Screenshots / Live demo link

![image](https://github.com/anuraghazra/github-readme-stats/assets/99110613/ba7e4f97-ee5e-4789-acab-c815225c184e)

**Comment by qwerty541** [comment, marked as answer]
Hey, @klpranaav!

Your readme file does not contain the code for displaying our stats card. There is code only for displaying top languages and streak cards and both of them works perfectly right now. You can check the screenshot below. So the solution for you is just adding the following string into your readme file:

```html
<img src="https://github-readme-stats.vercel.app/api?username=klpranaav" />
```

<img src="https://github-readme-stats.vercel.app/api?username=klpranaav" />

Also be aware that streak stats card does not belong to this repository, so in case of any problems with this card you should ask there: https://github.com/DenverCoder1/github-readme-streak-stats.

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/6a593993-353b-4402-932d-44da7e6e45e7)
